plone/app/controlpanel/interfaces.py

```diff
@@ -1,30 +1,20 @@
-from zope.interface import Attribute
-from zope.interface import Interface
+# -*- coding: utf-8 -*-
+from Products.CMFPlone.interfaces import controlpanel
 
 
-class IPloneControlPanelView(Interface):
-    """A marker interface for views showing a controlpanel.
+class IPloneControlPanelView(controlpanel.IPloneControlPanelView):
+    """BBB class
+    DONT USE UNLESS YOU RUN INTO BACKWARD COMPATIBILITY PROBLEMS
```

Comment:
@jensens does it make sense to add deprecations as per https://docs.plone.org/develop/styleguide/deprecation.html#enable-deprecation-warnings ? :thinking:

Reply:
Otherwise :100: thanks for doing it!!

Reply:
I am not sure if we need it at all with its dotted path.  Maybe it is enough to just do a deferred import?

Reply:
yup, that's what I would do :+1:

Reply:
I created at least an issue for now https://github.com/plone/plone.app.controlpanel/issues/67